boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -195,4 +195,18 @@
 			reg = <0x136000 DT_SIZE_K(24)>;
 		};
 	};
+
+	cpusec_rw_partitions: cpusec-rw-partitions {
+		compatible = "nordic,owned-partitions", "fixed-partitions";
+		status = "disabled";
+		perm-read;
+		perm-write;
+		perm-secure;
```

Comment:
This is not meant to be reserved in UICR, is it?
```suggestion
	partitions {
		compatible = "fixed-partitions";
```

Reply:
No, it is statically allocated.